src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.html

```diff
@@ -235,7 +235,12 @@
             <div class="form-group row">
               <label i18n
                      for="reportId"
-                     class="cd-col-form-label">Report ID</label>
+                     class="cd-col-form-label">
+              <ng-container i18n>Report ID</ng-container>
+              <cd-helper i18n-html
+                         html="A randomized UUID to identify a particular cluster over the course of several telemetry reports.">
+              </cd-helper>
+              </label>
```

Comment:
This is the smaller change necessary to get it working on octopus:
```suggestion
                     class="cd-col-form-label">Report ID
              <cd-helper i18n-html
                         html="A randomized UUID to identify a particular cluster over the course of several telemetry reports.">
              </cd-helper></label>
```
just need to move the `</label>` next to the previous block.

Reply:
Thank you @tspmelo I was going through the docs to figure out what I did wrong XD